test_pull_requests

```diff
@@ -351,492 +354,597 @@ def compose_bot_comment(prefix, comment_data={})
   updated_comment
 end
 
-# Dynamically extend GitHubAPI to add methods that don't
-# require a local git project
-module Hub
-  class GitHubAPI
+# Implements the cache operations API expected by Faraday::HttpCache
+# (<https://github.com/plataformatec/faraday-http-cache#usage-and-configuration>),
+# and implements logic to invalidate particular classes of cache
```

Comment:
`and` in the description of this class smells -- can we have the `FlushableStore` underlie but be separate from the `InvalidatingStore`?